evaluator.py

```diff
@@ -87,12 +96,12 @@ def run(self, instruments, data):
             results[indicator_name] = result
         return results
 
-def evaluate(results, indicator, **serialized):
+def evaluate(results, indicator, **fields):
```

Comment:
Is there any reason why we must rename this variable?
This will be a dict, instead of a list of str, so it is not just "fields", but a dict which maps fields to their values.

Reply:
Oh!  In the last version, I put serialize function in global which will lead the name be the same. Thus, I change the argument name.  It would be change.